src/snowflake/snowpark/dataframe.py

```diff
@@ -1617,10 +1619,14 @@ def drop_duplicates(self, *subset: Union[str, Iterable[str]]) -> "DataFrame":
         return df
 
     @df_to_relational_group_df_api_usage
+    @private_preview_parameter(version="1.15.0")
```

Comment:
I don't think it is required anymore since you have warnings right now?

Reply:
Will remove